english/eat-tacos/eat-delicious-tacos

```diff
@@ -0,0 +1,7 @@
+You found a place in the street, you were still hungry and you didn't have a lot of money. Then you decided to eat in that place.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.